### ISSUE TITLE
Update feature-detection code on demo

### ISSUE THREAD

**Issue by s3ththompson** [WebAssembly/webassembly.github.io#27]
Proposal:

```js
if (typeof Wasm !== "undefined" && Wasm.experimentalVersion < 0xc) {
    var exports = Wasm.instantiateModule(binary, info).exports;
    mergeMemory(exports.memory);
    applyMappedGlobals();
    return exports;
} else if (typeof WebAssembly !== "undefined") {
  var exports = new WebAssembly.Instance(new WebAssembly.Module(binary), info).exports;
  mergeMemory(exports.memory.buffer);
  applyMappedGlobals();
  return exports;
} else {
  // asm.js fallback
}
```

**Comment by lukewagner** [comment]
lgtm

**Comment by titzer** [comment]
Won't we also have to do version probing to distinguish 0xC, 0xD, and 0x1?

**Comment by lukewagner** [comment]
I was assuming this would be added (to the part where we compute the URL to fetch) with 0xD.

**Comment by titzer** [comment]
SGTM.

**Comment by s3ththompson** [comment]
I'll land this after #35 is finished.

**Comment by lukewagner** [comment]
Included in #37.
